factorial.rb

```diff
@@ -0,0 +1,9 @@
+class Factorial
+  def find(number)
+    raise ArgumentError, 'Number must be greater than or equal to 0.' if number.negative?
+
+    return 1 if number.zero?
+
+    (1..number).reduce(:*)
+  end
+end
```

Comment:
Good job for adding tests 💯👍

Please let's always read all requirements before we start building our project. 🤗

According to the [project requirements](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/testing/tdd_project.md), we are required to create a class with the name `Solver` implementing `#factorial`, `#reverse`, and `#fizzbuzz` methods in it. 🤓